fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java

```diff
@@ -890,6 +904,81 @@ public double estimateInnerRowCount(Statistics statistics, List<BinaryPredicateO
         return result;
     }
 
+    private double getPredicateSelectivity(PredicateOperator predicateOperator, Statistics statistics) {
+        Statistics estimatedStatistics = estimateStatistics(Lists.newArrayList(predicateOperator), statistics);
+        return estimatedStatistics.getOutputRowCount() / statistics.getOutputRowCount();
+    }
+
+    //  This estimate join row count method references ORCA.
+    //  use a damping method to moderately decrease the impact of subsequent predicates to account for correlated columns.
+    //  This damping only occurs on sorted predicates of the same table, otherwise we assume independence.
+    //  For example, given ANDed predicates (t1.a = t2.a AND t1.b = t2.b AND t2.b = t3.a) with the given selectivity:
+    //  (S1) t1.a = t2.a has selectivity 0.3
```

Comment:
What's the meaning of `S`?

Reply:
`t1.a = t2.a` is a predicate, not a selectivity?

Reply:
moved the S position